### PR TITLE
[WIP] Port TypeScript examples to Python

### PR DESCRIPTION
**Pull request by dstufft**
Ports the examples from TypeScript to Python.

- [x] ``application-load-balancer``
- [x] ``chat-app``
- [x] ``classic-load-balancer``
- [x] ``custom-resource``
- [ ] ``ecs``
- [ ] ``elasticbeanstalk``
- [x] ``lambda-cron``
  - This is ported, but broken due to https://github.com/awslabs/jsii/issues/411.
- [ ] ``my-widget-service``
- [ ] ``resource-overrides``
- [ ] ``static-site``
- [ ] ``stepfunctions-job-poller``

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: python/lambda-cron/app.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        lambdaFn = lambda_.Function(
+            self,
+            "Singleton",
+            code=lambda_.InlineCode(handler_code),
+            handler="index.main",
+            timeout=300,
+            runtime=lambda_.Runtime.PYTHON27,
+        )
+
+        # Run every dat at 6PM UTC
```
Comment (eladb) [inline python/lambda-cron/app.py]: Typo “dat”

**Comment by Doug-AWS** [comment]
I can probably take these and figure out how to port the Hello World tutorial in the guide. Tomorrow.

**Comment by dstufft** [comment]
Note these aren't fully working, waiting on an updated version of the Python CDK to be published so I can actually test them, but this should be pretty close.

**Comment by Doug-AWS** [comment]
No worries, I'm sure they are close enough for me to get the ball rolling.

**Comment by dstufft** [comment]
Other than the one marked as not working, these are all working now (and the one that's not working is ported correctly, there is just a JSII bug).

**Pull request closed without merging**